.github/workflows/build.yml

```diff
@@ -98,4 +98,4 @@ jobs:
           target: default
           arch: ${{matrix.arch}}
           tags: |
-            docker.io/hashicorp/${{env.repo}}:${{env.version}}
\ No newline at end of file
+            docker.io/hashicorp/${{env.repo}}:${{env.version}}
```

Comment:
I don't know what happened here, but I can confirm nothing changed here. 🤷‍♀️

Reply:
I think it's just because the new line at the end of the file has been removed. Adding it back should remove this from the diff!

Reply:
I double checked the editor and there was an newline and if you edit this file you will see that there's a newline 🙈 

I could be way off base or just simply not seeing something? :bushes:

Reply:
looks like the original yaml file was missing the newline, most folks editors normalize by adding a newline onto the end of most files anyway, so I'm fine with the appended line break, it'll likely happen sooner or later anyway.